### ISSUE TITLE
UI Customization options

### ISSUE THREAD

**Issue by khat33b** [react-native-picker/picker#47]
# Feature Request

There should be options to customize the colour of the down arrow in the dropdown, the background color of the modal which opens up and the font and size of the items.

## Why it is needed

It is needed for better customiztions and create better looking dropdowns.

## Possible implementation

Add a `style` option for `Item` and a `modalStyle` and `arrowColor` option for `Picker`.

### Code sample

```
<Picker
  selectedValue={this.state.language}
  style={{height: 50, width: 100}}
  modalStyle={{backgroundColor: 'white'}}
  arrowColor="red"
  onValueChange={(itemValue, itemIndex) =>
    this.setState({language: itemValue})
  }>
  <Picker.Item label="Java" value="java"  style={{ fontFamily: 'Helvetica' }}/>
  <Picker.Item label="JavaScript" value="js" />
</Picker>
```

**Comment by ehudley-medean** [comment]
definitely a much needed prop

**Comment by mrjeanp** [comment]
Why hasn't this been implemented already? you know, sometimes we use dark backgrounds in our apps

**Comment by Egizas** [comment]
This element is basically unusable with expo. No way to change font, color, arrow color or shape.

**Comment by sergiulucaci** [comment]
@EvanBacon any plan for this feature request?

**Comment by sadikyalcin** [comment]
+1 or add at least add dark theme support.

**Comment by hadnet** [comment]
I can't believe this s** is not implemented yet. It's just a background/text color for fGs..

**Comment by ilhamsa1** [comment]
waiting for update ... :D

**Comment by jsamr** [comment]
*To all those who resent, a little reminder.* This is a community repository (disclaimer, I am no maintainer). It means every consumer shares the responsibility to enhance the product he or she benefits freely. So, please avoid comments whining like everything  should be served on a golden plate and start contributing :sweat_smile:.

**Comment by hadnet** [comment]
> _To all those who resent, a little reminder._ This is a community repository (disclaimer, I am no maintainer). It means every consumer shares the responsibility to enhance the product he or she benefits freely. So, please avoid comments whining like everything should be served on a golden plate and start contributing 😅.

Community is not trash, it's not a place for you to just slap it together. If you do something, do it correctly, otherwise, f*off.

**Comment by 1nfinity-5starZ** [comment]
PR is open: https://github.com/react-native-community/react-native-picker/pull/121

Who can merge this ? It's already been approved

**Comment by chefk5** [comment]
Please guys merge this

**Comment by Naturalclar** [comment]
The feature to customize color of the dropdown icon has been implemented in v1.9.0 :tada:
note that the package is now under `@react-native-picker` scope, not under `@react-native-community`.
Please install the module from `@react-native-picker/picker`

**Comment by ericbaobao** [comment]
> The feature to customize color of the dropdown icon has been implemented in v1.9.0 🎉
> note that the package is now under `@react-native-picker` scope, not under `@react-native-community`.
> Please install the module from `@react-native-picker/picker`

how about the background color? I can't tell it is dropdown at all if I didn't click it

**Comment by francisco-j** [comment]
> > The feature to customize color of the dropdown icon has been implemented in v1.9.0 🎉
> > note that the package is now under `@react-native-picker` scope, not under `@react-native-community`.
> > Please install the module from `@react-native-picker/picker`
> 
> how about the background color? I can't tell it is dropdown at all if I didn't click it

not ideal, but i solved it by wraping the piker by a view and `backgroundColor`ing  it instead

**Comment by ThaJay** [comment]
Splitting up React-Native and ending oficcial support for loads of important default components was a huge mistake.

I do not want a random UI library. I just want the OS default with a usable api.
